docs/index.rst

```diff
@@ -4752,6 +4843,11 @@ And you can add more information to the predictions in a JSON format using:
      'prediction': u'Iris-versicolor',
      'distribution_unit': 'counts'}
 
+However, ``Boosted Trees`` don't have an associated confidence measure, so
+only the prediction will be obtained when applied to regressions.
+In classifications, adding the ``add_confidence`` or ``with_confidence``
+argument will also provide the probability of the prediction.
```

Comment:
since the caller of the function is probably knowing what ensemble kind she's using, perhaps you could consider having also `add_probability` or `with_probability`, and then you don't have to know about the kind of the ensemble in the function... or consider accepting both as synonyms... just a thought, not sure if that's more confusing than helpful...